Description: Fix -Wimplicit-function-declaration errors
 Fixes FTBFS with dpkg >= dpkg 1.22.6
Author: James Cowgill <jcowgill@debian.org>
Bug-Debian: https://bugs.debian.org/1066715
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gui/GtkGui.c
+++ b/gui/GtkGui.c
@@ -33,6 +33,7 @@
 
 #include "../libpcsxcore/plugins.h"
 #include "../libpcsxcore/cheat.h"
+#include "../libpcsxcore/cdrom.h"
 
 #include "MemcardDlg.h"
 #include "ConfDlg.h"
--- a/plugins/dfinput/pad.c
+++ b/plugins/dfinput/pad.c
@@ -22,6 +22,7 @@
 #include <sys/file.h>
 #include <time.h>
 #endif
+#include <sys/wait.h>
 
 #if SDL_VERSION_ATLEAST(2,0,0)
 int has_haptic;
--- a/plugins/dfxvideo/cfg.c
+++ b/plugins/dfxvideo/cfg.c
@@ -18,6 +18,7 @@
 #define _IN_CFG
 
 #include <sys/stat.h>
+#include <sys/wait.h>
 #include <unistd.h>
 #include <stdlib.h>
 
--- a/plugins/bladesio1/sio1.c
+++ b/plugins/bladesio1/sio1.c
@@ -31,6 +31,8 @@ void AboutDlgProc();
 void ConfDlgProc();
 #else
 #include <sys/stat.h>
+#include <sys/wait.h>
+#include <unistd.h>
 #endif
 
 #include "typedefs.h"
--- a/plugins/peopsxgl/stdafx.h
+++ b/plugins/peopsxgl/stdafx.h
@@ -62,6 +62,7 @@
 #include <unistd.h>
 #include <sys/stat.h>
 #include <sys/time.h>
+#include <sys/wait.h>
 #include <GL/gl.h>
 #include <GL/glx.h>
 #include <math.h> 
--- a/plugins/dfsound/stdafx.h
+++ b/plugins/dfsound/stdafx.h
@@ -51,6 +51,7 @@
 #define RRand(range) (random()%range)  
 #include <string.h> 
 #include <sys/time.h>  
+#include <sys/wait.h>
 #include <math.h>  
 
 #undef CALLBACK
--- a/plugins/dfcdrom/cdr.h
+++ b/plugins/dfcdrom/cdr.h
@@ -68,6 +68,7 @@ __private_extern__ char* PLUGLOC(char* t
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
+#include <sys/wait.h>
 #include <unistd.h>
 #include <pthread.h>
 #include <time.h>
